src/v3/plugins/github/relationalView.js

```diff
@@ -346,10 +346,32 @@ export class RelationalView {
     }
     for (const e of this.textContentEntities()) {
       const srcAddress = e.address();
-      for (const ref of parseReferences(e.body())) {
+      for (const {ref, refType} of parseReferences(e.body())) {
         const refAddress = refToAddress.get(ref);
         if (refAddress != null) {
-          this._addReference(srcAddress, refAddress);
+          switch (refType) {
+            case "basic":
+              this._addReference(srcAddress, refAddress);
+              break;
+            case "pairedWith":
+              if (refAddress.type !== N.USERLIKE_TYPE) {
+                throw new Error(
+                  `paired with non userlike: ${stringify(refAddress)}`
+                );
+              }
+              const userlike = this.userlike(refAddress);
+              if (userlike == null) {
+                throw new Error(
+                  `paired with nonexistent userlike: ${stringify(refAddress)}`
```

Comment:
This should not be an error.

Watch: Paired with @someonewhodoesnotexist.

Reply:
Okay: I see now how the code works. Thanks for improving the error message. :-)

Reply:
Done.